### PR TITLE
OCM-12269 | ci: fix with reducing the number of sg in id:73068

### PR DESCRIPTION
**Pull request by yuwang-RH**
[OCM-12269](https://issues.redhat.com//browse/OCM-12269) | ci: fix with reducing the number of sg in id:73068

### CODE REVIEW

**Comment by yuwang-RH** [comment]
@xueli181114 @jerichokeyne Could you help review this PR?Thanks.

File: tests/utils/profilehandler/handler.go [hunk 1]
```diff
@@ -1054,9 +1054,18 @@ func (ctx *profileContext) DestroyRHCSClusterResources(token string) error {
 		if err != nil {
 			errs = append(errs, err)
 		} else {
-			_, err = vpcService.Destroy()
+			_, err := vpcService.Destroy()
 			if err != nil {
-				errs = append(errs, err)
+				if strings.Contains(err.Error(), "DependencyViolation") {
+					Logger.Warnf("DependencyViolation, waiting for 15 minutes before retrying. The error is %v", err)
+					time.Sleep(15 * time.Minute)
```
Comment (xueli181114) [inline tests/utils/profilehandler/handler.go:1061]: Maybe it's better to use for loop rather than sleeping 15 mins?

Comment (yuwang-RH) [reply tests/utils/profilehandler/handler.go:1061]: The time it takes to destroy vpc is uncertain; sometimes it's longer, sometimes shorter, it almost takes about 20min ,so here we only want to retry once with an interval based on experience.
Considering it is a sporadic issue, although the probability of it occurring is a bit high recently, I will add comment to declare the time.sleep is just a workaround, will be removed in future if the dependency can be deleted before the vpc destroy. What do you think about it?

**Pull request merged**